### PR TITLE
https://issues.redhat.com/browse/ACM-13556

### PR DESCRIPTION
**Pull request by dockerymick**

### CODE REVIEW

File: release_notes/known_issues_governance.adoc [hunk 1]
```diff
@@ -23,17 +23,21 @@ Or consider a troubleshooting topic.
...
 
+////
 [#unable-to-log-out]
 == Unable to log out from Red Hat Advanced Cluster Management
 
 When you use an external identity provider to log in to {acm-short}, you might not be able to log out of {acm-short}. This occurs when you use {acm-short}, installed with IBM Cloud and Keycloak as the identity providers.
 
 You must log out of the external identity provider before you attempt to log out of {acm-short}. 
 
+//should this be in console?
```
Comment (mprahl) [inline release_notes/known_issues_governance.adoc]: Console seems like a good fit.

**Comment by dockerymick** [comment]
Thanks for the review @mprahl just made updates.

**Review by mprahl — APPROVED** [review]

File: release_notes/known_issues_console.adoc [hunk 2]
```diff
@@ -94,3 +94,10 @@ To work around the issue, zoom out by using your browser zoom controls, increase
 == _EditApplicationSet_ expand feature repeats
 
 When you add multiple label expressions or attempt to enter your cluster selector for your `ApplicationSet`, you might receive the following message repeatedly,  "Expand to enter expression". You can enter your cluster selection despite this issue.
+
+[#unable-to-log-out]
+== Unable to log out from Red Hat Advanced Cluster Management
```
Comment (jc-berger) [inline release_notes/known_issues_console.adoc]: Do we not want to use conrefs in titles? 

If so, disregard. 

```suggestion
== Unable to log out from {acm-short}
```

**Review by jc-berger — APPROVED** [review]
/lgtm

File: release_notes/known_issues_console.adoc (hunk 2 above)
Comment (dockerymick) [reply release_notes/known_issues_console.adoc]: Great catch!

**Comment by dockerymick** [comment]
@mprahl @jc-berger thanks for the reviews yall! Merging this PR

**Pull request merged**